dashboard/frontend/src/components/AppBar.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import {
+    withRouter
+} from 'react-router-dom'
+
+import { default as BaseAppBar } from 'material-ui/AppBar';
+import FlatButton from 'material-ui/FlatButton';
+import ActionDelete from 'material-ui/svg-icons/action/delete';
+import Dialog from 'material-ui/Dialog';
+
+import { deleteTfJob } from '../services';
+
+class AppBar extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isModalVisible: false
+        }
+    }
+
+
+    render() {
+        const actions = [
+            <FlatButton
+                label="CANCEL"
+                primary={true}
+                onClick={_ => this.setState({ isModalVisible: false })}
+            />,
+            <FlatButton
+                label="DELETE"
+                primary={true}
+                onClick={_ => {
+                    this.setState({ isModalVisible: false });
+                    this.deleteJob();
+                }}
+            />
+        ];
+
+        let rightMenu = null;
+        if (this.props.location.pathname != "/new") {
+            rightMenu = (
+                <div style={this.styles.rightMenu}>
+                    <FlatButton default={true} label="DELETE" icon={<ActionDelete />} style={this.styles.rightMenuButton} onClick={_ => this.setState({ isModalVisible: true })} />
+                </div>
+            );
+        }
+        return (
+            <div>
```

Comment:
in react 16 no need for wrapper

Reply:
Can we get rid of this per @Jimexist's comment?

Reply:
I think @Jimexist is mistaken here.  
JSX still needs a wrapper element, see https://github.com/airbnb/javascript/issues/1575 for example.

Reply:
you can wrap them in an array - to get rid of one level of DOM element. but it's minor issue I suppose.

Reply:
or if you use react 16.2 you can use `<>` and `</>`